virttest/virt_vm.py

```diff
@@ -659,6 +659,7 @@ def get_address(self, index=0):
             raise VMMACAddressMissingError(index)
         if self.ip_version == "ipv4":
             # Get the IP address from arp cache
+            del self.address_cache[nic.mac.lower()]
```

Comment:
Delete key "nic.mac.lower()" is not ok, in a test loop other test cases also use it. The error likes:
Unhandled KeyError: '9a:72:26:02:68:f0'    [context: (run) --> logging into 'virt-tests-vm1' --> (wait_for_get_address)]